### ISSUE TITLE
[Feature Request]: Volt::test('view.here') click to go

### ISSUE THREAD

**Issue by robsontenorio** [laravel-idea/plugin#846]
### Feature Description

It would be nice click to go working here. Same as #volt

<img width="358" alt="image" src="https://github.com/laravel-idea/plugin/assets/118955/d35b74e9-a96b-4c37-9dfa-8269177c2081">

**Comment by adelf** [comment]
It has the same syntax as Livewire::test() right? So, i should implement
assertHas, assert* livewire methods...

**Comment by robsontenorio** [comment]
@adelf 

 `Volt::test('view')` has same syntax `Livewire::test('view')`.

But my suggestion here is to implement click to go to view. Same you did at `Volt::route('view')`.

It has nothing to do with `assertXXXX` methods.


```php
Volt::test('view.here')  // <--- click to go to view here
  ->assert(...)
  ->assert(...);
```

**Comment by adelf** [comment]
So, you don't want completion for `->set()`, `->assertSet()`, and others?)

**Comment by robsontenorio** [comment]
@adelf 

That `set/assertXXX` already works, because Livewire source code uses `@mixin` that is parsed by PHPSTORM.

<img width="269" alt="image" src="https://github.com/laravel-idea/plugin/assets/118955/338f3745-469f-487f-8a99-0af835861441">



<br><br>


So, I only miss  **click to go view** at `Volt::test('view')`

**Comment by adelf** [comment]
<img width="356" alt="image" src="https://github.com/laravel-idea/plugin/assets/2818394/c21b8039-a823-4b97-a7a3-9ead73ca6aab">

<img width="340" alt="image" src="https://github.com/laravel-idea/plugin/assets/2818394/53c45601-603a-4ab1-953c-243c8bb08647">

:-) I can't promise, but will try to make this work too.

**Comment by robsontenorio** [comment]
@adelf ok, I love that ❤️

**Comment by adelf** [comment]
It's implemented. Will be in the next version. But you can have it right now by adding an `ide.json` file to the project.

```
{
    "$schema": "https://laravel-ide.com/schema/laravel-ide-v2.json",
    "completions": [
        {
            "complete": "livewireComponent",
            "condition": [
                {
                    "methodNames": ["test"],
                    "classFqn": ["\\Livewire\\Volt\\Volt"],
                    "parameters": [1]
                }
            ]
        }
    ]
}
```

**Comment by adelf** [comment]
Implemented in 7.4.2. Thanks.
